datadog_checks_dev/datadog_checks/dev/tooling/trello.py

```diff
@@ -50,13 +52,29 @@ def __init__(self, config):
             'Processes': '5e7910789f92a918152b700d',
             'Trace': '5c050640ecb34f0915ec589a',
             'Tools and Libraries': '5ab12740841642c2a8829053',
+            'Security': '5f314f0a364ee16ea4e78868',
         }
         self.progress_columns = {
             '55d1fe4cd3192ab85fa0f7ea': 'In Progress',  # INPROGRESS
             '58f0c271cbf2d534bd626916': 'Issues Found',  # HAVE BUGS
             '5d5a8a50ca7a0189ae8ac5ac': 'Awaiting Build',  # WAITING
             '5dfb4eef503607473af708ab': 'Done',
         }
+        self.__check_map_consistency(self.team_list_map, self.label_team_map, self.label_map)
```

Comment:
This should probably be done in a unit test instead

Reply:
IMO, this is the class responsibility is to ensure the consistency of the state of its internal fields (Not unit tests). `label_team_map` is used in testable.py but `label_team_map` should be a private field.

Anyway, I did not write unit tests because, as far as I know, there is no unit tests for `testable.py` :). 
I can add a new one if you want.

Reply:
I checked with ofek and there is no unit test for testable.py.